app/views/editions/secondary_nav_tabs/edit/editable/_local_transaction.html.erb

```diff
@@ -0,0 +1,96 @@
+<%= render partial: "editions/secondary_nav_tabs/edit/editable/components/title", locals: { edition: @resource } %>
+
+<section>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "LGSL code",
+    heading_level: 3,
+    font_size: "s",
+  } %>
+  <p>
+    <%= edition.lgsl_code.presence || "None added" %>
+  </p>
+</section>
+
+<%= render "govuk_publishing_components/components/input", {
+  label: {
+    text: "LGIL code",
+  },
+  id: "lgil",
+  name: "edition[lgil_code]",
+  value: edition.lgil_code,
+  heading_size: "s",
+  width:2,
+  class: "govuk-input--extra-letter-spacing",
```

Comment:
We should add an error message parameter here and link it to the error via the id `lgil_code` rather than the value used here.